zcutil/build.sh

```diff
@@ -65,18 +65,18 @@ fi
 set -x
 
 # If --enable-proton is the next argument, enable building Proton code:
```

Comment:
Can't we move this also to `CONFIGURE_FLAGS`?

Reply:
No, because it also requires the `WITH_PROTON` argument to the depends system.